Lib/fontTools/ttLib/ttGlyphSet.py

```diff
@@ -126,15 +145,60 @@ def draw(self, pen):
         how that works.
         """
         glyph, offset = self._getGlyphAndOffset()
+
+        if self.glyphSet.locationStack:
+            offset = 0  # Offset should only apply at top-level
```

Comment:
btw, I didn't see any mention of this detail about the offset only applying at top-level in the BE draft glyf1.md spec... perhaps it should.

Reply:
It's the undocumented behavior of the glyf table that toplevel glyphs are shifted to align their lsb to the hmtx table. Components never are. We need this here because VarComposite components are handled by ttGlyphSet whereas regular Composite components are handled in the glyf table.  So this is not really a glyf1-specific behavior, but I'll add a note there.

Reply:
Done:

Note: While it is the (undocumented?) behavior of the glyf table that glyphs loaded are shifted to align their LSB to that specified in the hmtx table, much like regular Composite glyphs, this does not apply to component glyphs being loaded as part of a VariableComposite glyph.